### ISSUE TITLE
The loss function of my network can be quickly reduced to 3, but basically it will not be reduced afterwards. And when I test on an image I can't get a correct pred

### ISSUE THREAD

**Issue by Bigwode** [shekkizh/FCN.tensorflow#102]
I set the learning rate =1e-5 and batch_size=2

**Comment by wmbai** [comment]
Have you solved the problem?

**Comment by goodbyeTimes** [comment]
I also have this problem

**Comment by PanlongXu** [comment]
Hi! How many times have you trained the model? By the way, can I have your test code?

**Comment by ShifengJin** [comment]
> I also have this problem

I also have this problem, have you solved the problem??

**Comment by chiyandetaotie** [comment]
I also have this problem. Have you solved the problem?
